src/tileset/wangset.rs

```diff
@@ -0,0 +1,101 @@
+use std::collections::HashMap;
+
+use xml::attribute::OwnedAttribute;
+
+use crate::{
+    error::Error,
+    properties::{parse_properties, Properties},
+    util::{get_attrs, parse_tag, XmlEventResult},
+    Result, TileId,
+};
+
+mod wang_color;
+pub use wang_color::WangColor;
+mod wang_tile;
+pub use wang_tile::{WangId, WangTile};
+
+/// Undocummented WangSet types
```

Comment:
What does this mean?

Reply:
That wangset types are not described on Tiled documentation, but they appear (and must be chosen) on the files. I didn't know how to describe them as I don't know what the differences are.

Reply:
Whoops, this should be fixed in the TMX format docs, which indeed forgets to mention the type entirely at https://doc.mapeditor.org/en/stable/reference/tmx-map-format/#wangset.

For the user, the various types are documented at https://doc.mapeditor.org/en/stable/manual/terrain/.